radiance.go

```diff
@@ -61,8 +63,9 @@ type ProxyStatus struct {
 // Radiance is a local server that proxies all requests to a remote proxy server over a transport.StreamDialer.
 // TODO: tunStatus need to be updated when TUN is active
 type Radiance struct {
-	srv         httpServer
-	confHandler configHandler
+	srv                  httpServer
+	confHandler          configHandler
+	activeProxyConfIndex *atomic.Int64
```

Comment:
This is only used to fetch the proxy location, so we could just save the proxy location right?